### PR TITLE
Ham mode should broadcast in plaintext and nodeinfo every 10 minutes

### PR DESCRIPTION
**Pull request by thebentern**

### CODE REVIEW

**Comment by m7xedhoward** [comment]
I believe (for amateurs in the UK) it doesn't need to be plain text as along as the encoding is publicly available. Is that correct?  Does that apply all other regions?

**Comment by caveman99** [comment]
> I believe (for amateurs in the UK) it doesn't need to be plain text as along as the encoding is publicly available. Is that correct? Does that apply all other regions?

This is not about encoding, the payload is encrypted with AES128 by default. We need to switch that functionality off, even for the UK.

File: src/modules/AdminModule.cpp [hunk 1]
```diff
@@ -609,9 +609,20 @@ void AdminModule::handleSetHamMode(const meshtastic_HamParameters &p)
     config.lora.override_duty_cycle = true;
     config.lora.tx_power = p.tx_power;
     config.lora.override_frequency = p.frequency;
+    // Set node info broadcast interval to 10 minutes
+    // For FCC minimum call-sign announcement
+    config.device.node_info_broadcast_secs = 600;
+
+    // Remove PSK of primary channel for plaintext amateur usage
+    auto primaryChannel = channels.getByIndex(channels.getPrimaryIndex());
```
Comment (caveman99) [inline src/modules/AdminModule.cpp:617]: grafik

**Review by caveman99 — CHANGES_REQUESTED** [review]

**Comment by caveman99** [comment]
What issue? These settings have been around for a long time, all this PR does is make it easier under the hood to enable them. For Ham operation the entire package content sent out needs to be plaintext readable. Even encryption with a default key is not permitted, let alone a custom key. Only way to use this legal as a ham is to turn encryption completely off, and this PR does exactly that.

From the UK i know that digital modes like Free-DV or DVB-S have been a point of discussion for a long time, as some saw the content as encrypted. I think this is what you are referring to, our Protobufs are public and the Transmissions are standardized, BUT there's additional real encryption which is not permitted and which we disable.

File: src/modules/AdminModule.cpp (hunk 1 above)
Comment (m7xedhoward) [reply src/modules/AdminModule.cpp:617]: If each option is available as a individual config option, encryption (on/default/off), tx power,  frequency,  duty. I don't to understand why a ham mode is required..

**Comment by m7xedhoward** [comment]
Indeed i was thinking of such examples.  after futher thought you are correct. Best to avoid all encryption, public or not for ham use.

File: src/modules/AdminModule.cpp (hunk 1 above)
Comment (mc-hamster) [reply src/modules/AdminModule.cpp:617]: It shows that we are putting an effort to comply with regional requirements for licensed use.

**Pull request merged**